mainwindow.cpp

```diff
@@ -418,9 +409,16 @@ void MainWindow::on_actionCopyToClipboard_triggered()
     if (!m_hashesTable)
         return;
     QModelIndexList indexes = m_ui->tableView_Hashes->selectionModel()->selectedIndexes();
-    // TODO: maybe copy everything if nothing is selected?
```

Comment:
I put an informative message saying : Nothing is selected. Please select rows/columns in the password table to copy them  to the clibpboard!"